### PR TITLE
sakura: fixed GLib-GIO-ERROR #21698

### PR DESCRIPTION
**Pull request by polyrod**
###### Motivation for this change
Bug Fix for issue; GLib-GIO-ERROR #21698

###### Things done
added wrapper to sakura binary wich sets path to gtk3-gsettings-schemata which provide
colorchooser fontchooser gui descriptions.

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/sakura/default.nix [hunk 1]
```diff
@@ -11,7 +11,11 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake perl pkgconfig ];
 
-  buildInputs = [ gtk3 vte pcre glib ];
+  buildInputs = [ makeWrapper gtk3 vte pcre glib ];
+
+  postInstall = ''
+        wrapProgram $out/bin/sakura --suffix XDG_DATA_DIRS : ${gtk3}/share/gsettings-schemas/${gtk3.name}/
+            '';
```
Comment (LnL7) [inline pkgs/applications/misc/sakura/default.nix]: whitespace

**Comment by FRidh** [comment]
please squash your commits so that we have only two left, one for sakura and one for chase.

**Comment by polyrod** [comment]
@FRidh @LnL7  hope its ok like this

**Review by LnL7 — APPROVED** [review]

**Comment by LnL7** [comment]
Thanks!

**Pull request merged**